test/typescript/unit-test.ts

```diff
@@ -228,6 +228,8 @@ voidReturn = channel.off(eventHandler);
 voidReturn = channel.on('message.new', eventHandler);
 voidReturn = channel.off('message.new', eventHandler);
 
+channel.sendMessage({ text: '' }); // send a msg without id
```

Comment:
could you add some text instead of empty? We might start validating it.

Reply:
this is just a type check

Reply:
yup, for consistency